versions/kruise/1.7.0/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+name: kruise
+description: Helm chart for kruise components
+version: 1.7.0
+appVersion: 1.7.0
+kubeVersion: ">= 1.18.0-0"
+icon: https://openkruise.io/img/openkruise-logo-bg.jpg
+keywords:
+  - openkruise
+  - kubernetes
+  - kruise
+  - workload
+  - statefulset
+  - sidecar
+  - job
+  - deployment
+  - cloneset
+home: https://openkruise.io
+sources:
+  - https://github.com/openkruise/kruise
+annotations:
+  artifacthub.io/changes: |
+    - "[Changed]: https://github.com/openkruise/kruise/blob/master/CHANGELOG.md"
+    - "[Security]: Fix potential security issues of dependent packages"
```

Comment:
how about add a change log about clusteripwithhostnetwork , plz refer to 
https://github.com/openkruise/charts/commit/7afba0dc8ff0f9f89d480cbca2c5040a7020e9dc